source/stylesheets/_variables.scss

```diff
@@ -1,8 +1,11 @@
 // colors
-$color-white: #ffffff;
-$color-black: #191919;
-$color-light: #f4f4f4;
-$color-red  : #ee0034;
+$color-white:      #ffffff;
```

Comment:
`#fff` 💅  ?

Reply:
As the other colors have 6 characters I prefer to stay with the pattern. But if u want performance we could put `white` hahaha

Reply:
As I said it's a 💅. It's up to you.